src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,362 @@
+    climberOrchestra.loadMusic("music/jeopardymusic.chrp");
+    // For simulation
+    // simulates the entire simulation, not just one arm
+    if (Robot.isSimulation()) {
+
+      simClimberGearbox = DCMotor.getKrakenX60(ClimberConstants.KClimberMotorsCount);
+      climberSimNormal =
+          new SingleJointedArmSim(
+              simClimberGearbox,
+              ClimberConstants.kGearRatio,
+              SingleJointedArmSim.estimateMOI(
+                  ClimberConstants.kClimberArmLength.in(Meters),
+                  ClimberConstants.kClimberMass.in(Kilograms)),
+              ClimberConstants.kClimberArmLength.in(Meters),
+              ClimberConstants.kClimberArmMinAngle.in(Radians),
+              ClimberConstants.kClimberArmMaxAngle.in(Radians),
+              true,
+              climberTargetAngle.in(Radians));
+      climberSimLifting =
+          new SingleJointedArmSim(
+              simClimberGearbox,
+              ClimberConstants.kGearRatio,
+              Math.pow(ClimberConstants.kClimberArmLength.in(Meters), 2)
+                  * DrivetrainConstants.ROBOT_MASS_KG,
+              ClimberConstants.kClimberArmLength.in(Meters),
+              ClimberConstants.kClimberArmMinAngle.in(Radians),
+              ClimberConstants.kClimberArmMaxAngle.in(Radians),
+              true,
+              ClimberConstants.kClimberRetractedSetpoint.in(Radians));
+
+      climbMech = new Mechanism2d(4, 2);
+      climbMechRoot1 = climbMech.getRoot("Climb Mech root", 3, 1);
+      climbMechRoot2 = climbMech.getRoot("Climb Mech root 2", 1, 1);
+      climbMechLigament1 =
+          climbMechRoot1.append(
+              new MechanismLigament2d(
+                  "Climb Mech Ligament 1",
+                  1,
+                  Radians.of(climberSimNormal.getAngleRads()).in(Degrees)));
+      climbMechLigament2 =
+          climbMechRoot2.append(
+              new MechanismLigament2d(
+                  "Climb Mech Ligament 2",
+                  1,
+                  Radians.of(climberSimNormal.getAngleRads()).in(Degrees)));
+      climbMechTargetLigament =
+          climbMechRoot1.append(
+              new MechanismLigament2d(
+                  "Climb Mech Target Ligament",
+                  0.8,
+                  climberTargetAngle.in(Degrees),
+                  10,
+                  new Color8Bit(255, 255, 0)));
+      SmartDashboard.putData("Climb Mech", climbMech);
+    }
+  }
+
+  public Command playJeopardy() {
+    return runOnce(
+        () -> {
+          climberOrchestra.play();
+        });
+  }
+
+  public Command stopJeopardy() {
+    return runOnce(
+        () -> {
+          climberOrchestra.stop();
+        });
+  }
+
+  public Command toggleJeopardy() {
+    return runOnce(
+        () -> {
+          if (climberOrchestra.isPlaying()) {
+            climberOrchestra.stop();
+          } else {
+            climberOrchestra.play();
+          }
+        });
+  }
```

Comment:
Scheduling these Commands will interrupt your climb Commands. You could either add the climberOrchestra.play()/stop() methods to another climb command, or you can use `Commands.runOnce` instead because it doesn't add the Climber subsystem as a requirement. Then you can tack it onto the Command mappings in RobotContainer, e.g. 
`OI.getTrigger(OI.Operator.RTrigger).onTrue(climber.climb().alongWith(climber.playJeopardy());`